VideoWeb/VideoWeb/ClientApp/src/app/shared/service-down-banner/service-down-banner.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="govuk-!-margin-top-3">
+  <div
+    class="govuk-notification-banner govuk-notification-banner--warning"
+    role="region"
+    aria-labelledby="govuk-notification-banner-title"
+    data-module="govuk-notification-banner"
+  >
+    <div class="govuk-notification-banner__header">
+      <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">Relase Notification</h2>
```

Comment:
typo here: should be `Release Notification`

perhaps we should use translations here? Presumably though the serviceDownBannerText is only in English

Reply:
Will add it any way just in case